components/eamxx/src/physics/mam/eamxx_mam_aci_process_interface.hpp

```diff
@@ -212,6 +212,7 @@ class MAMAci final : public scream::AtmosphereProcess {
       // for atmosphere
       compute_vertical_layer_heights(team, dry_atm_pre_, i);
       compute_updraft_velocities(team, wet_atm_pre_, dry_atm_pre_, i);
+      set_min_background_mmr(team, dry_aero_pre_, i); //dry_atm_pre_ is the output
```

Comment:
@singhbalwinder I don't recognize this line, could this be something I inadvertently added back during a rebase?

Reply:
Sorry, ignore this, I remember you adding a fix in the original SCREAM PR.